common/culture/cultures/orc.txt

```diff
@@ -496,9 +464,6 @@
 	}
 
 	laughting_skull = {
```

Comment:
Minor Typo:
```suggestion
	laughing_skull = {
```

Reply:
This feeling when fixing a typo may break more things in the future. I wouldn't fix this.